doc/api_assets/api.js

```diff
@@ -143,6 +143,7 @@
         const parentNode = el.target.parentNode;
 
         const flavorSelector = parentNode.querySelector('.js-flavor-selector');
+        const consoleSelector = parentNode.querySelector('.language-console');
```

Comment:
This should be moved below, there's no need to store this variable the code branches into the `else` block. There's probably no need for a variable at all since it's used once.
```suggestion
```

Reply:
Done